src/App.vue

```diff
@@ -252,6 +279,10 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return process.env.ABOUT_TEXT
   }
 
+  private get filingTitle (): string {
+    return this.isIncorporationFiling ? FilingNames.INCORPORATION_APPLICATION : FilingNames.DISSOLUTION_FILING
```

Comment:
Why not change `isIncorporationFiling` to `getFilingName: FilingNames` ?

This would support more types in the future than a boolean.

Reply:
Good idea, will do 👍

Reply:
Good fix here, refactored and applied in `EntityInfo` as well, in next commit.